vmaas_sync/send_messages.go

```diff
@@ -19,24 +19,25 @@ func sendReevaluationMessages() error {
 
 	ctx := context.Background()
 
-	for _, inventoryID := range inventoryIDs {
-		sendOneMessage(ctx, inventoryID)
-	}
+	sendMessages(ctx, inventoryIDs...)
```

Comment:
It would be better to control batch size for sending messages. Now it will send e.g. millions of messages in one batch.